src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToGrid/Grid.java

```diff
@@ -0,0 +1,59 @@
+package edu.greenblitz.tobyDetermined.commands.swerve.MoveToGrid;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.DriverStation;
+
+public class Grid {
+	private static Grid instance;
+	private Location selectedPos;
```

Comment:
Raz - after the small discussion yeterday (10.2.2023), "Pos" is not that clear. Maybe use Pose/Position. (In general throught the code)

Reply:
done